.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
we should update the two `.nvmrc` files in the repo to 14.13 as well

Reply:
Is there a string that you think would be better for nvm than "14"?

As far as I can tell (And I don't use nvm), there isn't a way to say >=14.13.0 in a way nvm understands. Some discussion here https://github.com/nvm-sh/nvm/issues/2562 . I don't want to force an old version of the 14 branch on nvm users, and 14.13.* is pretty old at this point.

Reply:
Yeah, that's unfortunate. I never actually looked into these files, just was aware of their existence. Leaving them both at `14` is probably best then.